### PR TITLE
Update blog-oauth2.jdl for JHipster v8

### PR DESCRIPTION
**Pull request by mraible**

### CODE REVIEW

**Review by DanielFran — APPROVED** [review]

**Comment by DanielFran** [comment]
@mraible you should target [v8 branch](https://github.com/jhipster/jdl-samples/tree/v8)

**Comment by mshima** [comment]
@DanielFran do you have admin permission at this repository? Please set v8 branch as default.

File: README.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
-# Sample JDL files for JHipster
+# Sample JDL files for JHipster v7
```
Comment (mraible) [inline README.md]: Should this say v8?

Comment (DanielFran) [reply README.md]: yes

Comment (mraible) [inline README.md]:
```suggestion
# Sample JDL files for JHipster v8
```

File: README.md [hunk 2]
```diff
@@ -1,6 +1,8 @@
-# Sample JDL files for JHipster
+# Sample JDL files for JHipster v8
 
-This repository is a showcase of JDL schema files that are generated from JDL Studio.
+For updated JHipster v8 samples you should go to [v8 branch](https://github.com/jhipster/jdl-samples/tree/v8).
```
Comment (mraible) [inline README.md]: @DanielFran I updated the title, but now this looks out of place. Should it point to a v7 branch?

**Comment by DanielFran** [comment]
> @DanielFran do you have admin permission at this repository? Please set v8 branch as default.

Done

We have now 3 branches:
- v8 (default)
- v7
- v6

File: README.md (hunk 2 above)
Comment (DanielFran) [reply README.md]: @mraible I believe we should remove it

Comment (DanielFran) [inline README.md]:
```suggestion
```

**Review by DanielFran — CHANGES_REQUESTED** [review]

**Review by DanielFran — APPROVED** [review]

**Pull request merged**